### DISCUSSION TITLE
How to create top child Menu?

### DISCUSSION THREAD

**Discussion by ewwink** [ObaraEmmanuel/Formation#29] [Q&A] [answered]
I want to create top menu with child

![image](https://github.com/ObaraEmmanuel/Formation/assets/760764/d2a35593-ceb3-4d45-bc44-146d1bc75cb0)

**Comment by ObaraEmmanuel** [comment, marked as answer]
To add a toplevel menu, you need to add a cascade type menu item:


![Screenshot 2024-03-13 191233](https://github.com/ObaraEmmanuel/Formation/assets/36537378/4c10786a-890d-411a-a322-8cbcad2a191b)

Then add the sub menu items under it. To add sub menu items just select the specific cascade menu then add the menu item of your choice and it will be inserted under it. You can also drag the items around to the right positions.

![Screenshot 2024-03-13 191346](https://github.com/ObaraEmmanuel/Formation/assets/36537378/2d4e3cc7-84ea-45bb-aa74-d6e979c7960d)

> Don't mind the the different icons, I am using a development version that will be out soon

**Comment by ewwink** [comment]
thank you
